Trippy/Utils/DateUtils.m

```diff
@@ -28,10 +28,15 @@ + (int)aheadSecondsFrom1970:(NSDate *)date aheadBy:(int)aheadBy {
     return round(seconds) + aheadBy; //
 }
 
-+ (NSArray *)secondsToHourMin:(int)seconds {
++ (struct TimeInHrMin)secondsToHourMin:(int)seconds {
     int hours = seconds / 3600;
     int minutes = (seconds - (hours * 3600)) / 60;
-    return @[[NSNumber numberWithInt:hours], [NSNumber numberWithInt:minutes]];
+    TimeInHrMin time = {.hours = hours, .minutes = minutes};
+    return time;
+}
+
++ (NSNumber *) hourMinToSeconds:(TimeInHrMin)time {
+    return [[NSNumber alloc] initWithInt:time.hours * 3600 + time.minutes * 60];
```

Comment:
PEMDAS, yes; but a good habit (for readability too) to wrap your operation pairs with parentheses.